BUILD

```diff
@@ -1,11 +1,75 @@
+load("@rules_python//python:defs.bzl", "py_runtime", "py_binary")
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+load("//:tools/psim.bzl", "psim_autocode")
+
+# Build flight software's gnc code as a library.
+cc_library(
```

Comment:
Who is the consumer of this library? Note that FSW can't really use this stuff because we're still going with PlatformIO, the Bazel transition wasn't too successful.

Reply:
We'll have a Python module build target that will consume the library.